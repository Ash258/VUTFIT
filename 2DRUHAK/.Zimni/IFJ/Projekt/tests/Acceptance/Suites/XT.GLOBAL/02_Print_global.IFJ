' Global variable writing

dim shared b as integer = 38

function foo() as integer
	print b;
	b = 12
	print b;
end function

function bar() as integer
	print b;
end function

scope
	print b;
	dim b as integer
	print b;
	b = foo()
	print b;
	b = bar()
end scope
