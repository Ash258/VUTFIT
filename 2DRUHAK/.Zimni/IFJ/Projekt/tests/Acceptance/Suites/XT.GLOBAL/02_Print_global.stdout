 38 0 38 12 0 12